rln/src/utils.rs

```diff
@@ -45,77 +47,44 @@ pub fn bytes_le_to_fr(input: &[u8]) -> (Fr, usize) {
     )
 }
 
-pub fn bytes_be_to_fr(input: &[u8]) -> (Fr, usize) {
-    let el_size = fr_byte_size();
-    (
-        Fr::from(BigUint::from_bytes_be(&input[0..el_size])),
-        el_size,
-    )
-}
-
+#[inline(always)]
 pub fn fr_to_bytes_le(input: &Fr) -> Vec<u8> {
     let input_biguint: BigUint = (*input).into();
     let mut res = input_biguint.to_bytes_le();
     //BigUint conversion ignores most significant zero bytes. We restore them otherwise serialization will fail (length % 8 != 0)
-    while res.len() != fr_byte_size() {
-        res.push(0);
-    }
-    res
-}
-
-pub fn fr_to_bytes_be(input: &Fr) -> Vec<u8> {
-    let input_biguint: BigUint = (*input).into();
-    let mut res = input_biguint.to_bytes_be();
-    // BigUint conversion ignores most significant zero bytes. We restore them otherwise serialization might fail
-    // Fr elements are stored using 64 bits nimbs
-    while res.len() != fr_byte_size() {
-        res.insert(0, 0);
-    }
+    res.resize(fr_byte_size(), 0);
     res
 }
 
+#[inline(always)]
 pub fn vec_fr_to_bytes_le(input: &[Fr]) -> Result<Vec<u8>> {
-    let mut bytes: Vec<u8> = Vec::new();
-    //We store the vector length
-    bytes.extend(u64::try_from(input.len())?.to_le_bytes().to_vec());
-
-    // We store each element
-    input.iter().for_each(|el| bytes.extend(fr_to_bytes_le(el)));
+    let mut bytes = Vec::with_capacity(8 + input.len() * fr_byte_size());
```

Comment:
could you please add comment why such size. Just for code readability in the future